src/python/pants/backend/python/subsystems/python_setup.py

```diff
@@ -127,6 +132,20 @@ def resolver_allow_prereleases(self):
   def resolver_blacklist(self):
     return self.get_options().resolver_blacklist
 
+  @property
+  def build(self):
+    return self.get_options().build
+
+  @property
+  def default_precedence(self):
+    precedence = Sorter.DEFAULT_PACKAGE_PRECEDENCE[:]
+    if self.build is True:
+      allow_builds(precedence)
```

Comment:
this call to `allow_builds` here doesn't seem necessary, afaict - since `SourcePackage` is always going to be in the default precedence?